tasks/build_tags.py

```diff
@@ -54,6 +54,7 @@
         "kubeapiserver",
         "kubelet",
         "netcgo",
+        "orchestrator",
```

Comment:
Just to confirm, this has no impact on the size of the binary? (additional k8s libs might be pulled)

Reply:
just build the agent on my machine from master and my branch: both binaries are 114MB